### DISCUSSION TITLE
What kinds of Chinese font is used?

### DISCUSSION THREAD

**Discussion by yuantao313** [logisim-evolution/logisim-evolution#1490] [General]
I launched this software and set the language to Chinese, but the toolbox showed a lot of "boxes", which is referred to the lack of font. Maybe my system does not have thois font. Can you help me?

**Comment by yuantao313** [comment]
![image](https://user-images.githubusercontent.com/57281450/180636777-71f76122-fd5f-4c0b-a3da-84b400937bf3.png)
"boxes" like this

**Reply by MarcinOrlowski** [reply]
Unfortunately this translation is not actively maintained and noone in the dev team speaks Chinese to answer your question. If you solved this yourself. please share the knowledge, but I started  the process or removing unmaintained translations completely. 

* #1503

**Reply by maehne** [reply]
Instead of removing unmaintained localisations, we should just ensure that appropriate fonts are used that cover all of them or at least document to what font needs to be switched. A bad localisation is a better starting point for a new localisation contributor than none.
